src/Component/spec/Symfony/Request/State/ProviderSpec.php

```diff
@@ -16,10 +16,13 @@
 use Pagerfanta\Pagerfanta;
 use PhpSpec\ObjectBehavior;
 use Psr\Container\ContainerInterface;
+use stdClass;
```

Comment:
Should be used directly as `\stdClass` as part of symfony coding style guideline.